debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -101,6 +104,9 @@ public BinlogReader(String name, MySqlTaskContext context) {
         client.registerLifecycleListener(new ReaderThreadLifecycleListener());
         if (logger.isDebugEnabled()) client.registerEventListener(this::logEvent);
 
+        Boolean filterDmlEventsByGtidSource = context.config().getBoolean(MySqlConnectorConfig.GTID_SOURCE_FILTER_DML_EVENTS);
```

Comment:
Technically, the `getBoolean(Field)` returns a `boolean` since the `Field` definition will have a default value that we can return if the configuration doesn't have a value for the same field. BTW, `getBoolean(String)` does return a `Boolean` object since the configuration may or may not have a value for the field with that name, and we aren't supplying a `Field` definition object.